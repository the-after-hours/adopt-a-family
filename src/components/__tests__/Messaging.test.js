import React from 'react';
import Messaging from '../Messaging';
import renderer from 'react-test-renderer';

describe('Messaging', () => {
  test('Messaging snapshot', () => {
    const component = renderer.create(<Messaging />);
    const tree = component.toJSON();

    expect(tree).toMatchSnapshot();
  });
});
