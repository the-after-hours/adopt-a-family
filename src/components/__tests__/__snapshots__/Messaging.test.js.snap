// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Messaging Messaging snapshot 1`] = `
<div>
  <h1>
    Messaging page
  </h1>
</div>
`;
